x/farming/keeper/reward.go

```diff
@@ -51,24 +33,16 @@ func (k Keeper) GetRewardsByFarmer(ctx sdk.Context, farmer sdk.AccAddress) (rewa
 // SetReward implements Reward.
 func (k Keeper) SetReward(ctx sdk.Context, stakingCoinDenom string, farmerAcc sdk.AccAddress, rewardCoins sdk.Coins) {
 	store := ctx.KVStore(k.storeKey)
-	// TODO: only rewardCoins
-	// 0x31 | StakingCoinDenomAddrLen (1 byte) | StakingCoinDenom | FarmerAddrLen (1 byte) | FarmerAddr -> ProtocolBuffer(sdk.Coins) RewardCoins
-	reward := types.Reward{
-		Farmer:           farmerAcc.String(),
-		StakingCoinDenom: stakingCoinDenom,
-		RewardCoins:      rewardCoins,
-	}
-	bz := k.cdc.MustMarshal(&reward)
+	bz := k.cdc.MustMarshal(&types.Reward{RewardCoins: rewardCoins})
```

Comment:
I think we need to marshal as `types.RewardCoins` type to unify with unmarshal logic